common/api/core-frontend.api.md

```diff
@@ -10921,6 +10922,8 @@ export namespace TileAdmin {
         optimizeBRepProcessing?: boolean;
         retryInterval?: number;
         tileExpirationTime?: number;
+        // @beta
+        tileStorage?: FrontendStorage;

```

Comment:
Why is this `@beta`? Is @itwin/object-storage-core open source? Is it publicly documented? I can find neither from [npmjs.org](https://www.npmjs.com/package/@itwin/object-storage-core).

Reply:
`@itwin/object-storage-core` lives here https://github.com/iTwin/object-storage

Reply:
package,json is missing "repository" pointing to this repo, so people cannot find it from npmjs.org.
Where is documentation? Does this package follow semantic versioning? Is it feasible to expect third-party developers to figure out how to use it on their own and rely on it to remain stable?

Reply:
fair point about documentation, @gdatbentley can you add instructions for Alicloud in NextVersion.md?
CC @Animagne @austeja-bentley